### ISSUE TITLE
chocolate recipe was hard to follow!

### ISSUE THREAD

**Issue by belinda-liu** [open-sauce/recipes#7]
i tried following the chocolate sauce recipe and it was disappointing! how is anyone supposed to enjoy the chocolate when getting mouthfuls of burnt wrapper?

steps to repro:

1. turn on stove (gas)
1. place full sized hershey bar (milk) on stove
1. wait for melting
1. attempt to consume

this set off my smoke alarm and has also ruined two (2) stove burners now. please advise. if it helps, i'm running a gentoo linux machine.

**Comment by xoebus** [comment]
Thank you for your feedback!

We've received a lot of feedback about our chocolate recipe over the past quarter. To everyone out there who has left us feedback: we hear you! We're going to take a step back and revisit our approach and provide a better experience for our customers.
